data/real_world/amsterdam_canals/H.txt

```diff
@@ -1,3 +1,3 @@
-1.248303947368415434e+00  0.000000000000000000e+00  1.212800310000000027e+05
-0.000000000000000000e+00  1.246897777777803196e+00  4.861667500000000000e+05
+3.794843999999982564e+00  0.000000000000000000e+00  1.212800310000000027e+05
```

Comment:
This seems to be more correct with scaling the map.

Reply:
How did you calculate this? We used our elementary linear algebra skills to make this matrix so I believe you immediately!

Reply:
Oh right!